scripts/predictions_to_shp.py

```diff
@@ -0,0 +1,57 @@
+import pickle
```

Comment:
zamiast tego potrzebny będzie skrypt rzeczywiście odpalający model na obrazie, więc nie wciągam go w tej formie.  O ile wiem to Cyprian i Piotrek mają teraz pracować nad takim skryptem